backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -292,13 +292,27 @@ export class ExperimentClientController {
    *            description: Log blob data
    */
   @Post('bloblog')
-  public blobLog(@Req() request: express.Request): Promise<Log[]> {
-    return new Promise((resolve) => {
+  public blobLog(@Req() request: express.Request): any {
```

Comment:
@YashilDepani We should not use any if we already know the type of it which was `Log` previously

Reply:
If I use any then the code is not allowing me to add `.catch ()` for the promise rejection

Reply:
@YashilDepani Can you look more into this to fix this?

Reply:
I was not able to find anything related to this. Just had a thought of doing `Promise<Log[] | any>`.